colossalai/shardformer/layer/_operation.py

```diff
@@ -110,10 +110,9 @@ def forward(ctx, input_, weight, bias, process_group, async_grad_allreduce):
         ctx.process_group = process_group
         ctx.async_grad_allreduce = async_grad_allreduce
 
+        output = torch.matmul(input_, weight.t())
```

Comment:
Do you compare these two implementation? Why is it faster than linear? It's counter-intuitive